src/admin/components/driver/DriverTableRow.tsx

```diff
@@ -41,20 +30,22 @@ export default function DriverTableRow({ user, index }: tableRowProp) {
       <td className="px-4 py-2">{user.ratings}</td>
       <td className={`px-4 py-2 `}>
         <span
-          className={`${user.account_status === ("Not Verified" || "not verified")
-            ? "text-white bg-red-500 py-2 px-4 rounded"
-            : "text-white bg-green-500 py-2 px-4 rounded"
-            }`}
+          className={`${
+            user.account_status === ("Not Verified" || "not verified")
```

Comment:
Use Enum data type for conditional classes.
`enum accountStatusStyles {
    verified= "text-white bg-red-500 py-2 px-4 rounded",
    notVerified= "text-white bg-green-500 py-2 px-4 rounded"
}`